handbook/engineering/web/index.md

```diff
@@ -44,6 +44,34 @@ Each issue should be as small as possible.
 Over the course of the iteration, we strive to close issues with small, iterative pull requests that we review and merge timely (generally a day).
 By the end of an iteration, all issues should be in the "Done" column and the board gets emptied.
 
+#### Iteration planning
+
+To ensure the team can be productive right from the start of an iteration we start planning for our next iteration as soon as a new iteration starts.
+
+1. First week of the iteration
+   1. The EM & PM will add tasks that should be considered for the iteration throughout the week. 
+      1. Tasks are added to the [Iteration planning scratchpad](https://docs.google.com/spreadsheets/d/1PKxOps4AWlqdI-H2eKSRzNhw4XT-oiwOpKF4UdN_1b8/edit); a spreadsheet we use to propose work items for the upcoming iteration
+      1. The PM will focus on adding items that are aligned with our roadmap so that we keep working towards our goals
+      1. The EM will focus on adding bugs and tech debt issues to ensure we maintain a high quality code base and product.
+   1. The EM & PM starts working on getting all tasks to a "Ready for dev" status.
+      1. A "Ready for dev" status means that all the necessary information has been capture or prepared so that the engineer taking up the issue can essentially start working on it immediately.
+         1. This does not involve figuring out implementation detail, as we do not tell engineers how to execute a task, but rather tell them the problem and make the requirements clear, and then ask them to figure out the best way to implement the solution.
+         1. We don't block on design dependencies for a task, but try to identify any dependency early on and communicate this requirement to the product designer.
+   1. The PM identifies and communicate design input needed from the Product Designer and works to prioritize it with them.     
+1. Second week of the iteration
+   1. By the Wednesday, the EM and PM should have ensured that:
+      1. All planned work have refined issues in GitHub
```

Comment:
Is there going to be a separate github board? Do we want a "next iteration" column of our current iteration board? A new column on the backlog board? I feel like a new column on the backlog board that's like "on deck" would be best, but open to other ideas.

Reply:
@Joelkw I think we keep the current iteration board clean and add a column to the backlog. Also don't think it's needed to introduce a new project board at this stage.